crowdin/messages.json

```diff
@@ -1832,6 +1833,7 @@
   "-197830563": "The ability to access a limited portion of clients’ sensitive information, our back-end code, or internal information on GitHub",
   "-611936805": "Exposed logs without sensitive information",
   "-1076255913": "Asunción | Our office – Paraguay | Deriv",
+  "-241792402": "Berlin – Our office | Deriv",
```

Comment:
```suggestion
  "-241792402": "Berlin | Our office | Deriv",
```

Reply:
I don't have access to the content file, I just suggest this based on the other translations.

Reply:
https://docs.google.com/document/d/16Nq-k26sUzovcjRyyej-pfxdcvgInQJ6NvQQxgkTM_o/edit# 
That's the file I followed, should I keep it as it is?

Reply:
Actually, I don't have access y to google docs.